### DISCUSSION TITLE
use xmake to create a cross-compile sdk

### DISCUSSION THREAD

**Discussion by fneddy** [xmake-io/xmake#5072] [Q&A]
Hi,

I was wondering if it is possible to use xmake to create a local cross-compile sdk. 

my idea:

use add_requires in a toolchain config and install the cross-compiler into the sdk folder.
is something like this possible?

**Comment by waruqi** [comment]
like this? https://github.com/xmake-io/xmake/blob/master/tests/projects/package/toolchain_muslcc/xmake.lua
